Example/Emission/AppDelegate.m

```diff
@@ -1,45 +1,55 @@
 #import "AppDelegate.h"
 
+#import <Emission/AREmission.h>
 #import <Emission/ARComponentViewController.h>
 #import <Emission/ARTemporaryAPIModule.h>
+#import <Emission/ARSwitchBoardModule.h>
 
-#import <React/RCTBridge.h>
 #import <React/RCTUtils.h>
+#import <TargetConditionals.h>
+
+#ifdef TARGET_OS_SIMULATOR
+#define ENABLE_DEV_MODE
+#endif
```

Comment:
You can comment this out to use the release JS bundle that’s bundled in the app.